adoc/advanced_topics_proxy_quickstart.adoc

```diff
@@ -162,7 +163,7 @@ For more information, see <<at.manager.proxy.register.saltclients>>.
 [[at.manager.proxy.run.pattern]]
 === Install the [path]``suma_proxy`` pattern
 
-Make sure the [path]``suma_proxy`` pattern version 2.5.1.3 or later is installed using the following command on the proxy as root:
+Either on the sever select the [package]``pattern_suma_proxy`` package for installation, or make sure the [path]``suma_proxy`` pattern is installed using the following command on the proxy as root:
```

Comment:
You can probably remove the 'either' to make this make more sense. Also typo on "server" ;)

Reply:
Oops, thanks!